### DISCUSSION TITLE
!data/01_raw/* should not be in the default .gitignore

### DISCUSSION THREAD

**Discussion by grofte** [kedro-org/kedro#3635] [Idea]
If I make a new Kedro project the lines 

```
# keep also the example dataset
!data/01_raw/*
```

are in .gitignore file. At the same time you have written in the README that users should not delete any of the lines you have added to .gitignore. I feel like this is going to hurt a lot of newbies that accidentally add a big dataset to git and then struggle to get rid of it. Even for the example with spaceflights I think it's a terrible idea to add the data to git.

PS I have not installed the example.

PPS I could not find it in your codebase or I would have written a PR. At the same time it's not a bug, just a terrible choice.

**Comment by noklam** [comment]
The starters are not in `kedro` template, it is cloned via `cookiecutter` and has its own repository: https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pandas

**Reply by noklam** [reply]
Thank you for raising this, I actually don't know this exists. I agree that this doesn't seem to be necessary to have this line even we need to include the data in git.

> Even for the example with spaceflights I think it's a terrible idea to add the data to git.
I think the idea is to have something that users can quickly get started and it hasn't caused any issues in the past. 
Some alternatives that I can think of, but I don't think they are better:
1. Ask user to download data separately first.
2. Add a new node to download data from the nodes. This creates more complications, i.e. re-run the pipeline means downloading the data multiple times. It also not fits into Kedro DataCatalog concept, using something like an "APIDataset" to download data would be more appropriate, but they are unnecessarily complicated for a "Get Started" tutorial.

What do you think? This is separate from the proposal though, I agree `!data/01_raw/*` is not necessary.

**Reply by DimedS** [reply]
I agree, thank you @grofte , better to fix that

**Reply by datajoely** [reply]
Why don't we explicitly allowlist the files needed in `01_raw` for the starter and remove the wildcard?

**Reply by grofte** [reply]
But you already have a bunch more files when you init the example compared to the just a plain init. Can't you have the `!data/01_raw/specific_file` only occur in a .gitignore that comes with the example? And then a different .gitignore for the plain init?

**Reply by grofte** [reply]
Why not put the data in a public cloud bucket and then have the pipeline pull it from there? Isn't one of the selling points of the Kedro catalog that it's easy to do something like that? And then you get to demo it in the tutorial. There's even three files so you could host them on AWS, GCP, and Azure.

**Reply by grofte** [reply]
Can you load from an URL? Because then you could put it in a Gist. The CSVs at least.
